README.md

```diff
@@ -147,6 +148,8 @@ This will provide access to API Gateway at [http://localhost:3001](http://localh
 
 Additionally, this ties in with the `serverless webpack` plugin which will ensure that your lambdas are re-built when changes are detected.
 
+Running the following command will enable integration with local AWS services: `S3=true SQS=true npm run sls`
```

Comment:
Instead of making the dev set env variables in their command, just put them into the npm script in package.json

something like
`"offline:with-export": "S3=true SQS=true npm run offline"`

Reply:
done with https://github.com/nasa/earthdata-search/pull/1630/commits/cff58f3c43be9715ef408555ff4aec0a87169318

Reply:
I think you need to run `docker-compose up` beforehand.  I could create a convenience script that runs both like:
```js
"backend": "docker-compose up && npm run offline:with-export"
```
or add docker-compose up to offline:with-export like
```js
"offline:with-export": "docker-compose up && S3=true SQS=true npm run offline"
```
Just let me know what you prefer.

Reply:
In cmr-ordering we use these scripts to start the queue

```
    "start": "serverless offline start",
    "prestart": "npm run start:queue",
    "start:queue": "docker-compose up -d",
    "stop:queue": "docker-compose down"
```

Reply:
updated with https://github.com/nasa/earthdata-search/pull/1630/commits/083a54f4ec43c3918288ebaee1568e5ccb6d026d